apps/sms/views/inbox/js/inbox.js

```diff
@@ -114,14 +113,10 @@ var InboxView = {
     MessageManager.on('message-received', this.onMessageReceived.bind(this));
     MessageManager.on('threads-deleted', this.onThreadsDeleted.bind(this));
 
+    Drafts.init();
     Drafts.on('deleted', this.onDraftDeleted.bind(this));
     Drafts.on('saved', this.onDraftSaved.bind(this));
 
-    InterInstanceEventDispatcher.on(
-      'drafts-changed',
-      this.renderDrafts.bind(this, true /* force update */)
```

Comment:
__nit:__ I think we can remove ```force``` parameter support from ```renderDrafts``` (and unit tests if we have any for this parameter) to keep code simpler if we don't use it anymore, what do you think?